lib/Controller/PublicViewController.php

```diff
@@ -104,6 +104,8 @@ public function publicIndexForEmbedding(string $token):PublicTemplateResponse {
 		$csp->addAllowedFrameAncestorDomain('*');
 		$response->setContentSecurityPolicy($csp);
 
+		$this->initialStateService->provideInitialState($this->appName, 'is_embed', true);
```

Comment:
is this used?

Reply:
Yes, needed to target only the embed page so that we don't hide the header on the public page as well. Otherwise the only other solution I found was to use the `has()` CSS selector (to select based on child presence criteria), but it's [not fully available yet](https://caniuse.com/css-has).

Reply:
Found it: ``input#initial-state-calendar-is_embed``. I looked for js code that fetches the state.

I guess that works until we change the internals of initial state

Reply:
I'll propose https://github.com/nextcloud/calendar/pull/5251#issuecomment-1557406604 to avoid this